.github/workflows/tool-niricfg.yaml

```
@@ -0,0 +1,77 @@
+            org.opencontainers.image.source=${{ github.server_url }}/${{ github.repository }}
+            org.opencontainers.image.revision=${{ github.sha }}
+
+      - name: Build executable
+        run: |
+          set -e
+          ./make.py niricfg
+
+      - name: Upload niricfg artifact
+        uses: ./.github/actions/upload-image
+        with:
+          image: ${{ steps.image.outputs.name }}
```

actionlint (1.7.11)

[error] property "image" is not defined in object type {}

(expression)